package.json

```diff
@@ -7,18 +7,21 @@
     "url": "https://github.com/Esya/oasis-wallet.git"
   },
   "scripts": {
-    "start": "react-scripts -r ./internals/scripts/set-app-version-env.js -r @cypress/instrument-cra start",
+    "clean": "rm -rf build/ .parcel-cache",
+    "parcel": "parcel --port 3000 --target web --dist-dir build",
```

Comment:
This script name is annoying for debugging
`yarn parcel --help` runs `parcel --port 3000 --target web --dist-dir build --help`

Reply:
renamed to `parcel:dev` and `parcel:prod`